src/styles/movies.scss

```
@@ -0,0 +1,300 @@
+.container {
+    padding-top: 100px;
+    color: #fff;
+
+    @media (max-width: 720px) {
+        padding-top: 70px;        
+    }
+
+    a {
+        color: #1CB5E0;
+        text-decoration: none;
+    }
+
+    .wrapper {
+        padding: 0 6px;
+        margin-bottom: 2rem;
+    }
+
+    .card, 
+    .card *,
+    .card + * {
+        -webkit-transition: all .2s ease-in-out;
+        -moz-transition: all .2s ease-in-out;
+        -ms-transition: all .2s ease-in-out;
+        -o-transition: all .2s ease-in-out;
+        transition: all .2s ease-in-out;
+
+        @media (max-width: 480px) {
+            transition: none;
+        }
+    }
+
+    .info_panel {
+        padding: 20% 5% 0;
+        color: #fff;
+        background: transparent;
```

You're just modifying the background color
```suggestion
        background-color: transparent;
```